types/common.ts

```
@@ -0,0 +1,423 @@
+  slot: string;
+  setDisplayAd?: (display: boolean) => void;
+  adClient?: string;
+}
+
+export interface AdArticleProps {
+  isDisplay?: boolean;
+  slot: string;
+}
+
+export interface AdContentProps {
+  children: React.ReactNode;
```

ESLint

[error] 'React' is not defined.

(no-undef)